### PR TITLE
Implement Institution whitelist

### PR DESCRIPTION
**Pull request by DRvanR**

### CODE REVIEW

**Comment by DRvanR** [comment]
:pray:  @rjkip

File: src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/Command/AddToWhitelistCommand.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace Surfnet\StepupMiddleware\CommandHandlingBundle\Identity\Command;
+
+use Surfnet\StepupMiddleware\CommandHandlingBundle\Command\AbstractCommand;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class AddToWhitelistCommand extends AbstractCommand
+{
+    /**
+     * @Assert\Type(type="array")
```
Comment (rjkip) [inline src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/Command/AddToWhitelistCommand.php]: You could add:

```
@Assert\All({
    @Assert\NotBlank,
    @Assert\Type("string")
})
```

Same goes for the other whitelist commands.

File: src/Surfnet/Stepup/Identity/Collection/InstitutionCollection.php [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+use ArrayIterator;
+use Broadway\Serializer\SerializableInterface;
+use IteratorAggregate;
+use JsonSerializable;
+use Surfnet\Stepup\Exception\RuntimeException;
+use Surfnet\Stepup\Identity\Value\Institution;
+
+/**
+ * @
```
Comment (rjkip) [inline src/Surfnet/Stepup/Identity/Collection/InstitutionCollection.php]: This can probably be removed.

**Comment by rjkip** [comment]
Wicked. Ship it! :+1:

File: src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/Command/AddToWhitelistCommand.php (hunk 1 above)
Comment (DRvanR) [inline src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/Command/AddToWhitelistCommand.php]: do did done

File: src/Surfnet/Stepup/Identity/Collection/InstitutionCollection.php (hunk 2 above)
Comment (DRvanR) [inline src/Surfnet/Stepup/Identity/Collection/InstitutionCollection.php]: done

**Pull request merged**